cmake/configs/default.cmake

```diff
@@ -393,3 +393,4 @@ hunter_config(zookeeper VERSION 3.4.9-p2)
 hunter_config(tacopie VERSION 2.4.0-h1)
 hunter_config(cpp_redis VERSION 3.5.0-h1)
 hunter_config(IF97 VERSION 2.1.2)
+hunter_config(EnumGroup VERSION 0.0.1)
```

Comment:
"No newline at end of file" warning:
* http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file

Reply:
Done

Reply:
Got a conflict here.

Hint: The probability of the conflict will be much lower if you will keep an alphabetical order.